reconcile/utils/jinja2/utils.py

```diff
@@ -32,8 +32,18 @@ def __init__(self, msg: Any):
 
 
 @cache
-def compile_jinja2_template(body: str, extra_curly: bool = False) -> Any:
-    env: dict = {}
+def compile_jinja2_template(
+    body: str,
+    extra_curly: bool = False,
+    trim_blocks: bool = False,
+    lstrip_blocks: bool = False,
+    keep_trailing_newline: bool = False,
+) -> Any:
+    env: dict[str, Any] = {
+        "trim_blocks": trim_blocks,
+        "lstrip_blocks": lstrip_blocks,
+        "keep_trailing_newline": keep_trailing_newline,
+    }
```

Comment:
env will be reset if `extra_curly` is `True`, these params not needed for extra curly?

Reply:
Fixed in [21cbd8c](https://github.com/app-sre/qontract-reconcile/pull/4471/commits/21cbd8c42d919b392e62e952df472148abd838da)